cmd/buckler/main.go

```diff
@@ -812,11 +815,21 @@ func main() {
 			}
 			opts.Init.Master = a
 		}
-		err := c.Initialize(opts.Init.Master)
+		disasterKey, err := c.Initialize(opts.Init.Master)
 		bail(err)
 
 		fmt.Printf("SHIELD core unlocked successfully.\n")
 
+		if disasterKey != "" {
+			fmt.Printf("@R{BELOW IS YOUR DISASTER KEY FOR RECOVERING FIXED-KEY BACKUPS.}\n")
+			fmt.Printf("@R{SAVE THIS IN A SECURE LOCATION.}\n")
+			fmt.Printf("----------------------------------------------------------------\n")
+			fmt.Printf("@Y{" + c.SplitKey(disasterKey, 64) + "}")
+			fmt.Printf("\n----------------------------------------------------------------\n")
+		} else {
```

Comment:
Is it possible for Initialize() to return a nil err and an empty disaster key? If it is, we should fix that.

Reply:
It should not be possible (at least from the API standpoint always a bad, fail, or they key will exist) these checks were more to ensure future api changes dont introduce a bug that isnt caught quickly.